planning/autoware_mission_planner_universe/src/mission_planner/mission_planner.cpp

```diff
@@ -122,15 +122,24 @@ void MissionPlanner::publish_processing_time(
   pub_processing_time_->publish(processing_time_msg);
 }
 
-void MissionPlanner::publish_pose_log(const Pose & pose, const std::string & pose_type)
```

Comment:
The publish_pose_log currently only prints initial and goals, but I think it is intended to be a more generic function.
It is ok to leave this function as it is and print the api name at the beginning of on_*. Please comment if you have opinios.